api/app/routers/land_change/dist_alerts/dist_alerts.py

```diff
@@ -0,0 +1,346 @@
+        ...,
+        title="Start Date",
+        description="Must be either year or YYYY-MM-DD date format.",
+        pattern=DATE_REGEX,
+        examples=["2020", "2020-01-01"],
+    )
+    end_date: str = Field(
+        ...,
+        title="End Date",
+        description="Must be either year or YYYY-MM-DD date format.",
+        pattern=DATE_REGEX,
+        examples=["2023", "2023-12-31"],
+    )
+    intersections: List[Literal["driver", "natural_lands"]] = Field(
+        ..., min_length=0, max_length=1, description="List of intersection types"
+    )
+
+
+@router.post(
+    "/analytics",
+    response_class=ORJSONResponse,
+    response_model=DataMartResourceLinkResponse,
+    status_code=202,
+)
+def create(
+    *, data: DistAlertsAnalyticsIn, request: Request, background_tasks: BackgroundTasks
+):
+    # Convert model to JSON with sorted keys
+    payload_dict = data.model_dump()
+
+    # Convert to JSON string with sorted keys
+    payload_json = json.dumps(payload_dict, sort_keys=True)
+
+    # Generate deterministic UUID from payload
+    resource_id = str(uuid.uuid5(uuid.NAMESPACE_DNS, payload_json))
+
+    # Store payload in /tmp directory
+    payload_dir = PAYLOAD_STORE_DIR / resource_id
+    metadata_data = payload_dir / "metadata.json"
+    analytics_data = payload_dir / "data.json"
+
+    link = DataMartResourceLink(
+        link=f"{str(request.base_url).rstrip('/')}/v0/land_change/dist_alerts/analytics/{resource_id}"
+    )
+
+    if metadata_data.exists() and analytics_data.exists():
+        return DataMartResourceLinkResponse(data=link, status=AnalysisStatus.saved)
+
+    if metadata_data.exists():
+        return DataMartResourceLinkResponse(data=link, status=AnalysisStatus.pending)
+
+    payload_dir.mkdir(parents=True, exist_ok=True)
+    metadata_data.write_text(payload_json)
+    background_tasks.add_task(do_analytics, file_path=payload_dir)
+    return DataMartResourceLinkResponse(data=link, status=AnalysisStatus.pending)
+
+
+class AnalysisStatus(str, Enum):
+    saved = "saved"
+    pending = "pending"
+    failed = "failed"
+
+
+class DistAlertsAnalytics(StrictBaseModel):
+    result: Optional[dict] = {  # column oriented for loading into a dataframe
+        "__dtypes__": {
+            "country": "str",
+            "region": "int",
+            "subregion": "int",
+            "alert_date": "int",
+            "confidence": "int",
+            "value": "int",
+        },
+        "country": ["BRA", "BRA", "BRA"],
+        "region": [1, 1, 1],
+        "subregion": [12, 12, 12],
+        "alert_date": [731, 733, 733],
+        "confidence": [2, 2, 3],
+        "value": [38, 5, 3],
+    }
+    metadata: Optional[dict] = None
+    message: Optional[str] = None
+    status: AnalysisStatus
+
+    model_config = {
+        "from_attributes": True,
+        "validate_by_name": True,
+    }
+
+
+class DistAlertsAnalyticsResponse(Response):
+    data: DistAlertsAnalytics
+
+
+async def do_analytics(file_path):
```

Comment:
so we'll still need to put this in a background task next sprint?

Reply:
Currently, it runs in a [fastapi BackgroundTask](https://fastapi.tiangolo.com/tutorial/background-tasks/). You can see where it's being used in the [path operation function](https://github.com/wri/project-zeno-data-infra/pull/25/files#diff-61d3d763d429935208b5b0c2abd954414f7a633882c482acff82876cc0c6278dR200) above. As a result, the server sends the response immediately and works on the analytics without blocking.

Obviously, this isn't a sustainable solution. So, yes, we'll design a more robust solution in the next sprint.